src/py/CMakeLists.txt

```diff
@@ -22,27 +22,25 @@
 # THE SOFTWARE.
 #####################################################################################
 
-option(MIGRAPHX_ENABLE_PYTHON "Enable python bindings" ON)
+add_compile_definitions($<$<COMPILE_LANGUAGE:CXX,C>:MIGRAPHX_ENABLE_PYTHON>)
```

Comment:
This should only be added to the driver: `target_compile_definitions(driver MIGRAPHX_ENABLE_PYTHON=1)`